config/packages/prod/monolog.yaml

```diff
@@ -30,6 +30,13 @@ monolog:
             token: '%env(SLACK_TOKEN)%'
             channel: 'dev-kit-logs'
             bot_name: '@@dev-kit-logger'
-            icon_emoji: ':ghost:' # 👻
             level: debug
             include_extra: true
+        slack:
+            type: slack
+            token: '%env(SLACK_TOKEN)%'
+            channel: 'dev-kit-logs'
```

Comment:
I have trouble to understand the config.

Why is it again the `dev-kit-logs` channel and the dev-kit-logger ?
But then there is a `channels` key ?

Reply:
That's a Slack channel. I think we could archive `#ci` and put the dev-kit logs in dev-kit-logger

Reply:
When I read this, I understood that the `#ci` channel was replaced by a `#automerge` channel.

Now I understand that with this config:
- The Slack_errors key will listen all the logs and post them in the `dev-kit-logs` slack channel.
- The Slack key will listen all the logs from the `automerge` monolog channel and post them in the `dev-kit-logs` slack channel.

I'm surely have a bad understanding because it would lead to duplicate logs...

Reply:
> The slack_errors key will listen all the logs and post them in the dev-kit-logs slack channel.

`slack_errors` is nested inside other handlers, the top-level one being of type `fingers_crossed`, and set with to be triggered with `error`. It will be only be triggered if there is an error or above beforehand.

The only case where you could have double posting is when we log an error or above in the `automerge` monolog channel.

Reply:
Ok. I see.

So now the only question I have is: Should we split the slack log by context (so having a dev-kit-logs for every errors and keeping/renaming the ci channel for the automerge warning) or should we have everything on dev-kit-logs like you did ?

`ci` channel was public, which could allow someone which is not in the `contributor` team to help us.

Reply:
:thinking: good point… I can change to `ci`, and then we can say we should have one slack channel per monolog channel, because calls to `this->logger->whatever` are voluntary while `slack_errors` is meant for things we didn't plan for like exceptions.

Reply:
We can change the strategy later if we decide too. But I think keeping the two channel is better ATM.
Plus it's BC 😁